src/factories/Axis.ts

```diff
@@ -20,6 +20,12 @@ module n3Charts.Factory {
       var vis: D3.Selection = this.factoryMgr.get('container').axes;
 
       this.createAxis(vis);
+
+      this.eventMgr.on('zoom.' + this.key, this.softUpdate.bind(this));
+    }
+
+    softUpdate() {
```

Comment:
I see, maybe we could add a little comment there - because I feel like I am forgetting all this small details.